Resources/Prototypes/DeltaV/Entities/Objects/Misc/rubber_stamp.yml

```diff
@@ -11,6 +11,20 @@
   - type: Sprite
     sprite: DeltaV/Objects/Misc/stamps.rsi
     state: stamp-psychologist
+
+- type: entity
+  name: pharmacist's rubber stamp
+  parent: RubberStampBase
+  id: RubberStampPharmacist
+  suffix: DO NOT MAP
```

Comment:
Worth considering leaving this stamp in the pharmacist office in case there isn't one aboard; so prescriptions can be officially stamped.